dispatch/static/manager/src/js/components/inputs/WidgetSelectInput.js

```diff
@@ -18,12 +22,15 @@ class WidgetSelectInputComponent extends React.Component {
   }
 
   render() {
+    const results = this.props.cWidgets ? R.keys(this.props.cWidgets) : this.props.widgets.zones[this.props.zoneId]
```

Comment:
I'd prefer to be a bit more verbose and call it something like `this.props.compatibleWidgets`